src/liblsquic/lsquic_packet_out.c

```diff
@@ -615,7 +615,7 @@ lsquic_packet_out_split_in_two (struct lsquic_mm *mm,
     struct stream_rec **new_srecs, **srecs = local_arr;
     struct stream_rec *srec;
     unsigned n_srecs_alloced = sizeof(local_arr) / sizeof(local_arr[0]);
-    unsigned n_srecs, max_idx, n, nbytes;
+    unsigned n_srecs, max_idx = 0, n, nbytes;
```

Comment:
Unnecessary initialization, but no fast way to simplify the code.  Make it MSVC-only?

Reply:
It looks like necessary initialization.  On line #656 it's used without being initialized.

Reply:
It gets initialized on the first pass of the loop (the left side of `||`):

```
        if (n_srecs == 0 || srecs[max_idx]->sr_len < srec->sr_len)
            max_idx = n_srecs;
```